### PR TITLE
Enable "Updates" section on organization

### PR DESCRIPTION
**Pull request by xdamman**
Fixes https://github.com/opencollective/opencollective/issues/2914

Allow admins to enable the "Updates" section on the profile page of their organization.

Also changed the "Post Update" button to "Preview Update" to avoid confusion since it is actually not posting it, but it goes to another page where you still need to post it (we ran into this ourselves)

### CODE REVIEW

**Comment by xdamman** [comment]
@mhulet can you check why the e2e tests don't pass?
Here is the error:

File: components/edit-collective/EditCollectiveUpdates.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
+import { Flex } from '@rebass/grid';
+import { set, cloneDeep, pick } from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useMutation } from 'react-apollo';
```
Comment (kewitz) [inline components/edit-collective/EditCollectiveUpdates.js]:
```suggestion
import { useMutation } from '@apollo/react-hooks';
```

File: components/edit-collective/EditCollectiveUpdates.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+const messages = defineMessages({
+  title: {
+    id: 'updates',
+    defaultMessage: 'Updates',
+  },
+  mainDescription: {
+    id: 'EditCollective.Updates.description',
+    defaultMessage:
+      'Updates is a way to keep your community posted on your progress. Once enabled, a new "Updates" section will be added to your profile page and a dedicated page will be created on {updatesLink}.',
```
Comment (Betree) [inline components/edit-collective/EditCollectiveUpdates.js]: **[:ice_cream: Details & personal preferences]** For me the most important thing about this feature is that it allows you to send a real **newsletter** (it's not just a blog post), I think we should somehow mention it.

```suggestion
      'Updates is a way to keep your community posted on your progress. They are sent by email to all you contributors and followers. Once enabled, a new "Updates" section will be added to your profile page and a dedicated page will be created on {updatesLink}.',
```

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>

Comment (xdamman) [reply components/edit-collective/EditCollectiveUpdates.js]: Agreed. Good suggestion. Let's merge that :-)
Thank you for helping in this PR, really appreciated!

**Comment by xdamman** [comment]
Any idea why the lang check fails? What's the process for this?

**Comment by Betree** [comment]
It's because the languages files are not up to date (see https://github.com/opencollective/opencollective-frontend/pull/3567/checks?check_run_id=524532975). You can fix that with a `npm run build:langs`.

**Comment by xdamman** [comment]
I did that before but just did it again to be sure. Still the same test failing.

**Comment by Betree** [comment]
It's passing now. The reason why you had to do it again is because we updated a default message in https://github.com/opencollective/opencollective-frontend/pull/3567/commits/b6d8f5b3d6ba9e2331a2d4a0154af2460fdf398c.

image

**Comment by Betree** [comment]
All green, this is going live. Thanks @xdamman!

**Pull request merged**